packages/squiggle-lang/src/rescript/Distributions/GenericDist/GenericDist_Types.res

```diff
@@ -50,11 +52,15 @@ module Operation = {
 
   type toFloatArray = Sample(int)
 
+  type toString = 
+  | ToString
+  | ToSparkline(int)
+
   type fromDist =
     | ToFloat(toFloat)
     | ToDist(toDist)
     | ToDistCombination(direction, arithmeticOperation, [#Dist(genericDist) | #Float(float)])
-    | ToString
+    | ToString(toString)
```

Comment:
two different types have constructors have the same name `ToString` (tho the `ToString` of `toString` is nullary and the `ToString` of `fromDist` is unary). I'm surprised the compiler let you do that, it must be the difference in arity. I also can't think of better names. 

Wanted to flag this for discussion (if alternative naming that you like comes to mind I think that'd be great!), but I don't think it'll cause friction.

Reply:
Yea, agreed it's kind of strange. Will consider going forward, but nothing comes to mind now.